src/sentry/static/sentry/app/views/settings/organizationMembers/organizationMemberDetail.jsx

```diff
@@ -142,16 +155,71 @@ class OrganizationMemberDetail extends AsyncView {
     });
   };
 
+  removeAuthenticator = (userId, authId) => {
```

Comment:
We've been trying to start moving api calls into functions e.g. https://github.com/getsentry/sentry/blame/master/src/sentry/static/sentry/app/actionCreators/account.jsx#L27

First argument should be `this.api` (the `disconnectIdentity` func should be fixed) because we attempt to abort api requests when the parent component is unmounted.